src/GraphBLAS-sharp/MatrixAndVector.fs

```diff
@@ -37,32 +39,56 @@ and [<AbstractClass>] Vector<'a when 'a : struct and 'a : equality>() =
     abstract Length: int
     abstract AsArray: 'a[]
 
-    abstract Item: Mask1D<'a> -> Vector<'a> with get, set
-    abstract Item: int * int -> Scalar<'a> with get, set
-    abstract Item: Mask1D<'a> -> Scalar<'a> with set
+    abstract CreateMask: bool -> Mask1D
```

Comment:
We can make 2 properties: `abstract Mask: Mask1D option` and `abstract Complemented: Mask1D option`. Hence, we won't need parentheses around the method call and could remove  static members from Mask1D/Mask2D. (Therefore, mask parameter in methods should be an option type).